api/tasks.go

```diff
@@ -791,11 +791,34 @@ func (wc *WaitConfig) Copy() *WaitConfig {
 	return nwc
 }
 
+type ChangeScript struct {
+	Path        *string        `mapstructure:"path" hcl:"path"`
```

Comment:
Sorry for missing this in the previous review, but I think a more accurate naming here would be this:
```suggestion
	Command        *string        `mapstructure:"command" hcl:"path"`
```

It's a small difference, but script implies a text file, whereas `Exec` will run other things, such as binaries. It will also fail to run scripts without shebang for example.

Another benefit is that it would also be more consistent with the [`check -> command`](https://www.nomadproject.io/docs/job-specification/check#command) option.

Reply:
ok, I updated across the codebase.